docs/best-practices.md

```diff
@@ -3,32 +3,8 @@ Best Practices
 
 This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
 
-
-## Custom Component
```

Comment:
I definitely see what you're saying, but I think a simple link to the "Custom Component" best practices section from the custom component should suffice. The thinking behind grouping best practices is allowing us to share suggestions / ideas that aren't necessarily core concepts in their own distinct section and sectioned by level (IE, component level, app level, etc.) But just my opinion and I could also see each best practices section living in its parent. :)